react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -86,6 +92,11 @@ class ConnectionStatsTable extends Component {
          */
         resolution: PropTypes.object,
 
+        /**
+         * The region of the media server.
```

Comment:
Maybe there needs to be a distinction made between region and serverRegion, at least in the docs.